src/Orleans.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
```

Comment:
@xiazen... Ok, will not arg, again... I wonder if you will hold this change to everyone every time they open Visual Studio with newer version, including core team... Anyway, consider done for the sake to finish this once and for all... We are looking for a problem that doesn't exist...